get_cpu_process/get_cpu_process.py

```diff
@@ -0,0 +1,48 @@
+import psutil
+# https://thispointer.com/python-check-if-a-process-is-running-by-name-and-find-its-process-id-pid/
+
+
+def is_running(provided_process_name: str) -> bool:
+    """
+    Takes the name of a process and
+    returns True if it is running,
+    False if it isn't"""
```

Comment:
It seems to me that docstring looks more beautiful - the text is separated from the literal
```
def is_running(provided_process_name: str) -> bool:
    """
    Takes the name of a process and
    returns True if it is running,
    False if it isn't
    """
```

Reply:
Done 👍